modules/preparing-an-inital-cluster-deployment-for-mce-connected.adoc

```diff
@@ -0,0 +1,287 @@
+    namespace: openshift-local-storage
+  spec:
+    installPlanApproval: Automatic
+    name: local-storage-operator
+    source: redhat-operators
+    sourceNamespace: openshift-marketplace
+----
++
+[NOTE]
+====
+After creating all the manifests, your Filesystem must display as follows:
+
+.Example Filesystem
+
+[source,terminal]
+----
+<assets_directory>
+    ├─ install-config.yaml
+    ├─ agent-config.yaml
+    └─ /openshift
+        ├─ mce_namespace.yaml
+        ├─ mce_operatorgroup.yaml
+        ├─ mce_subscription.yaml
+        ├─ lso_namespace.yaml
+        ├─ lso_operatorgroup.yaml
+        └─ lso_subscription.yaml
+----
+====
+
+. Create the `agent.iso` image by running the following command:
+
++
+[source,terminal]
+----
+$ openshift-install agent create image --dir <assets_directory>
+----
+
+. Once the image is ready , boot the target machine and wait for the installation to complete.
+
+. To monitor the installation, run the following command:
++
+[source,terminal]
+----
+$ openshift-install agent wait-for install-complete --dir <assets_directory>
+----
+
++
+[NOTE]
+====
+To configure a fully functional hub cluster,you must create the following manifests and manually apply them by running the command `$ oc apply -f <manifest-name>`.
+The order of the manifest creation is important and where required, the waiting condition is displayed.
+====
+
+. For the PVs that are required by the AI service, create the following manifests:
+
++
+[source,yaml]
+----
+  apiVersion: local.storage.openshift.io/v1
+  kind: LocalVolume
+  metadata:
+   name: assisted-service
+   namespace: openshift-local-storage
+  spec:
+   logLevel: Normal
+   managementState: Managed
+   storageClassDevices:
+     - devicePaths:
+         - /dev/vda
+         - /dev/vdb
+       storageClassName: assisted-service
+       volumeMode: Filesystem
+----
++
+You can use the following command to check the availability of the PVs:
++
+[source,terminal]
+----
+$ oc wait localvolume -n openshift-local-storage assisted-service --for condition=Available --timeout 10m
+----
+NOTE: The `devicePath` is just an example.
+
+. Create a manifest for an MCE instance.
++
+.Example MultiClusterEngine.yaml
+
++
+[source,yaml]
+----
+  apiVersion: multicluster.openshift.io/v1
+  kind: MultiClusterEngine
+  metadata:
+    name: multiclusterengine
+  spec: {}
+----
+
+. Create a manifest to enable the AI service.
++
+.Example agentserviceconfig.yaml
++
+[source,yaml]
+----
+  apiVersion: agent-install.openshift.io/v1beta1
+  kind: AgentServiceConfig
+  metadata:
+    name: agent
+    namespace: assisted-installer
+  spec:
+   databaseStorage:
+    storageClassName: assisted-service
+    accessModes:
+    - ReadWriteOnce
+    resources:
+     requests:
+      storage: 10Gi
+   filesystemStorage:
+    storageClassName: assisted-service
+    accessModes:
+    - ReadWriteOnce
+    resources:
+     requests:
+      storage: 10Gi
+----
+
+. Create a manifest to deploy spoke clusters.
```

Comment:
The following manifest could be useful to deploy _subsequently_ spoke clusters

Reply:
This is a procedural step that starts with a verb as per our style guidelines, I have added the word subsequently to the step.